src/controllers/vscodeWrapper.ts

```diff
@@ -337,4 +337,33 @@ export default class VscodeWrapper {
     public get onDidChangeConfiguration(): vscode.Event<vscode.ConfigurationChangeEvent> {
         return vscode.workspace.onDidChangeConfiguration;
     }
+
+    /**
+     * Called when there's a change in the extensions
+     */
+    public get onDidChangeExtensions(): vscode.Event<void> {
+        return vscode.extensions.onDidChange;
+    }
+
+    /**
+     * Gets the Azure Account extension
+     */
+    public get azureAccountExtension(): vscode.Extension<any> {
+        return vscode.extensions.getExtension(Constants.azureAccountExtensionId);
+    }
+
+    /**
+     * Returns true when the Azure Account extension is installed
+     * but not active
+     */
+    public get azureAccountExtensionActive(): boolean {
+        return this.azureAccountExtension && this.azureAccountExtension.isActive;
+    }
+
+    /**
+     * Returns whether an azure account is signed in
+     */
+    public get isAccountSignedIn(): boolean {
+        return this.azureAccountExtension.exports.status === 'LoggedIn';
+    }
```

Comment:
Should probably be safe and check for azureAccountExtension being undefined as well